cmd/server/app/options/options.go

```diff
@@ -293,6 +304,25 @@ func (o *ProxyRunOptions) Validate() error {
 		}
 	}
 
+	// Validate leasing parameters. All must be empty or have a value.
+	if o.LeaseLabelSelector != "" || o.LeaseDuration != 0 || o.LeaseGCInterval != 0 || o.LeaseRenewalInterval != 0 || o.LeaseNamespace != "" {
+		if o.LeaseLabelSelector == "" {
+			return fmt.Errorf("LeaseLabelSelector cannot be empty when leasing system is enabled")
+		}
+		if o.LeaseNamespace == "" {
+			return fmt.Errorf("LeaseNamespace cannot be empty when leasing system is enabled")
+		}
+		if o.LeaseDuration == 0 {
```

Comment:
Are we ok with it being negative?

Reply:
Probably not. Switched the expressions to be `<= 0`